src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -1,1243 +1,1251 @@
-            {
-                return rawInputOutputData;
-            }
-            else if (rawInputOutputData == null)
-            {
-                return "(null)";
-            }
-            else
-            {
-                // Azure Storage uses UTF-32 encoding for string payloads
-                return "(" + Encoding.UTF32.GetByteCount(rawInputOutputData) + " bytes)";
-            }
-        }
-
-        internal string GetExceptionTrace(string rawExceptionData)
-        {
-            if (rawExceptionData == null)
-            {
-                return "(null)";
-            }
-            else
-            {
-                return rawExceptionData;
-            }
-        }
-
-        /// <inheritdoc/>
-        Task<HttpResponseMessage> IAsyncConverter<HttpRequestMessage, HttpResponseMessage>.ConvertAsync(
-            HttpRequestMessage request,
-            CancellationToken cancellationToken)
-        {
-            return this.HttpApiHandler.HandleRequestAsync(request);
-        }
-
-        internal static string ValidatePayloadSize(string payload)
-        {
-            // The payload gets written to Azure Table Storage and to Azure Queues, which have
-            // strict storage limitations (64 KB). Until we support large messages, we need to block them.
-            // https://github.com/Azure/azure-functions-durable-extension/issues/79
-            // We limit to 60 KB (UTF-32 encoding) to leave room for metadata.
-            if (Encoding.UTF32.GetByteCount(payload) > 60 * 1024)
-            {
-                throw new ArgumentException("The size of the JSON-serialized payload must not exceed 60 KB of UTF-32 encoded text.");
-            }
-
-            return payload;
-        }
-
-#if !FUNCTIONS_V1
-        /// <summary>
-        /// Tags the current Activity with metadata: DurableFunctionsType, DurableFunctionsInstanceId, DurableFunctionsRuntimeStatus.
-        /// This metadata will show up in Application Insights, if enabled.
-        /// </summary>
-        internal static void TagActivityWithOrchestrationStatus(OrchestrationRuntimeStatus status, string instanceId, bool isEntity = false)
-        {
-            // Adding "Tags" to activity allows using Application Insights to query current state of orchestrations
-            Activity activity = Activity.Current;
-            string functionsType = isEntity ? "Entity" : "Orchestrator";
-
-            // The activity may be null when running unit tests, but should be non-null otherwise
-            if (activity != null)
-            {
-                string statusStr = Enum.GetName(status.GetType(), status);
-                activity.AddTag("DurableFunctionsType", functionsType);
-                activity.AddTag("DurableFunctionsInstanceId", instanceId);
-                activity.AddTag("DurableFunctionsRuntimeStatus", statusStr);
-            }
-        }
-#endif
-    }
-}
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Would it be possible to undo the whitespace diffs here? It makes it difficult to remove the changes in this file.

Reply:
Not sure what is happening here. I don't see this problem on my local checkout - which shows a normal diff.

Reply:
If that doesn't work, maybe the quickest workaround would just be to add a brief GitHub PR comment next to where the largest changes are so we know where to look.

Reply:
I think it worked, the diff looks normal now.